### PR TITLE
Hotfix/timeline update

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/HRIN-525

### CODE REVIEW

File: web/themes/custom/hoeringsportal/templates/field/field--dynamic-block-field--node-project-timeline.html.twig [hunk 1]
```diff
@@ -53,12 +53,12 @@
       <h3 class="mt-0 pt-4">{{ label }}</h3>
       <div class="project-timeline-period row">
         <div class="col-2">
-          <label>{{ 'Project start'|t }}</label>
-          <div>{{ items[0].content[0]['#node'].field_project_start.date.getTimestamp()|format_date('hoeringsportal_date') }}</div>
+          <label>{{ 'Start'|t }}</label>
+          <div>{{ items[0].content[0]['#node'].field_project_start.date.getTimestamp()|format_date('long') }}</div>
```
Comment (rimi-itk) [inline web/themes/custom/hoeringsportal/templates/field/field--dynamic-block-field--node-project-timeline.html.twig:57]: For consistency we should use one of the existing `hoeringsportal_` formats or create a new one if need be.

Comment (martinyde) [reply web/themes/custom/hoeringsportal/templates/field/field--dynamic-block-field--node-project-timeline.html.twig:57]: Hmm, the long format is added by drupal core on install, and fits the design perfectly, so what you are implying is really that all config provided by drupal core should be prefixed with sitename before taking it to use.

Comment (rimi-itk) [reply web/themes/custom/hoeringsportal/templates/field/field--dynamic-block-field--node-project-timeline.html.twig:57]: We [recently started using our own custom formats](https://github.com/itk-dev/hoeringsportal/commit/03da8574484b587035ec7420fbb4492c32889a4d) for all displays of dates and times in our custom code. This helps keep things clear and gives an overview of how (and in how many ways) we display times on the site.

Comment (martinyde) [reply web/themes/custom/hoeringsportal/templates/field/field--dynamic-block-field--node-project-timeline.html.twig]: Fixed

**Pull request merged**